src/KahanSummation.jl

```diff
@@ -113,4 +114,26 @@ function sum_kbn(A)
     s - c
 end
 
+function sum_kbn(f,A)
```

Comment:
Can we merge the two functions?  i.e. define `sum_kbn(A) = sum_kbn(identity, A)`?

Reply:
how is that done? how should I modify latest PR

Reply:
oh -- I see~